app-network/mihomo-party/spec

```diff
@@ -0,0 +1,5 @@
+VER=1.5.6
+SRCS="tbl::https://github.com/mihomo-party-org/mihomo-party/archive/refs/tags/v$VER.tar.gz"
```

Comment:
Do not use automatically packaged tarballs. See our [Styling Manual](https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#yuan-ma-wen-jian).